### ISSUE TITLE
Switch SHIFT key behavior back? Thor's idea?

### ISSUE THREAD

**Issue by xpalaum** [decentraland/explorer#254]
Community feedback on the change released not positive in general:
![image.png](https://images.zenhubusercontent.com/5d91dca0703cca0001efbe30/8beb8d24-a596-48af-acbb-295696ad9576)

We can bring the original behavior back.

Alternatively, we could try:
![image.png](https://images.zenhubusercontent.com/5d91dca0703cca0001efbe30/df607221-cd90-4c1c-a183-a489e2871a5f)

**Comment by BrianAmadori** [comment]
Design take-aways:

- Starting from walk state can be annoying because you have to wait for the transition to run each time you change direction (Already toyed with this mechanic many times in the past and didn't work).

- Inverting the shift again will not fix the scenes that require long jumps, because the long jumps depend upon movement speed. So this is a matter of movement speed, not the shift key behaviour.

- Fast movement speed breaks the avatar animation.

As we talked earlier, let's wait for more feedback before we take action on this. Its common to face initial rejection with this type of decisions that cripple previously attained advantages.

**Comment by xpalaum** [comment]
@BrianAmadori: is the default speed now exactly the same as the run speed we had before?

Facing initial rejection is not my worry. I am worried because we pushed a change without proper communication and decision making on our side, and I am not sure we aren't creating more problem than what we solved. For example, before I could already do long jumps from a platform (from idle position) just by pressing and holding SHIFT, then pressing W and SPACE. It was not super usable, but it was possible. It was also scene creator's choice to require long jumps for their scenes, something they chose to do based on previous behavior anyway.

![image.png](https://images.zenhubusercontent.com/5d91dca0703cca0001efbe30/6f9520fe-2d03-4775-b48b-d792a7687fa9)
